types/paypal-rest-sdk/index.d.ts

```diff
@@ -0,0 +1,761 @@
+        cb?: CallbackFunction<null>): void;
+    function deleteExternalRefund(
+        id: string,
+        trxid: string,
+        config: http.RequestOptions | CallbackFunction<null>,
+        cb?: CallbackFunction<null>): void;
+    function generateNumber(
+        config: http.RequestOptions | CallbackFunction<NumberResponse>,
+        cb?: CallbackFunction<NumberResponse>): void;
+    function get(
+        id: string,
+        config: http.RequestOptions | CallbackFunction<InvoiceResponse>,
+        cb?: CallbackFunction<InvoiceResponse>): void;
+    function list(
+        data: QueryParameters | http.RequestOptions | CallbackFunction<ListResponse>,
+        config?: http.RequestOptions | CallbackFunction<ListResponse>,
+        cb?: CallbackFunction<ListResponse>): void;
+    function qrCode(
+        id: string,
+        height: number,
+        width: number,
+        config: http.RequestOptions | CallbackFunction<QrResponse>,
+        cb?: CallbackFunction<QrResponse>): void;
+    function recordPayment(
+        id: string,
+        data: PayRequest,
+        config: http.RequestOptions | CallbackFunction<null>,
+        cb?: CallbackFunction<null>): void;
+    function recordRefund(
+        id: string,
+        data: RefundRequest,
+        config: http.RequestOptions | CallbackFunction<null>,
+        cb?: CallbackFunction<null>): void;
+    function remind(
+        id: string,
+        data: EmailRequest | http.RequestOptions | CallbackFunction<null>,
+        config?: http.RequestOptions | CallbackFunction<null>,
+        cb?: CallbackFunction<null>): void;
+    function search(
+        data: SearchRequest,
+        config: http.RequestOptions | CallbackFunction<ListResponse>,
+        cb?: CallbackFunction<ListResponse>): void;
+    function send(
+        id: string,
+        config: http.RequestOptions | CallbackFunction<ListResponse>,
+        cb?: CallbackFunction<null>): void;
+    function update(
+        id: string,
+        data: Invoice,
+        config: http.RequestOptions | CallbackFunction<InvoiceResponse>,
+        cb?: CallbackFunction<InvoiceResponse>): void;
+}
+export namespace refund {
+    function get(
+        id: string,
+        config: http.RequestOptions | CallbackFunction<RefundResource>,
+        cb?: CallbackFunction<RefundResource>): void;
+}
+export namespace sale {
+    function get(
+        id: string,
+        config: http.RequestOptions | CallbackFunction<PaymentResponse>,
+        cb?: CallbackFunction<PaymentResponse>): void;
+    function refund(
+        id: string,
+        data: RefundRequest | http.RequestOptions | CallbackFunction<RefundResource>,
+        config?: http.RequestOptions | CallbackFunction<RefundResource>,
+        cb?: CallbackFunction<RefundResource>): void;
+}
+// TODO: Strong Typing stops here
+export namespace billingAgreement {
+    function billBalance(id: string, data: any, config?: any, cb?: any): void;
```

Comment:
Shouldn't these be returning `any`, instead of `void` if they're untyped? (Since I assume many _do_ return values)